coalib/bearlib/aspects/Formatting.py

```diff
@@ -0,0 +1,378 @@
+from coalib.bearlib.aspects import Root, Taste
+
+
+@Root.subaspect
+class Formatting:
+    """Formatting refers to the visual appearance of source code.
```

Comment:
You tend to often create this redundancy: `<aspect> refers to...`, `<taste> refers to...`
Just:

```python
@Root.subaspect
class Formatting:
    """
    The visual appearance of source code.
    """
    ...
```